tests/cli/test_output_option.py

```diff
@@ -232,24 +233,29 @@ def test_explicit_inputs_can_be_in_inputs(renku_cli, client, subdirectory):
     assert plan.inputs[0].position is not None
 
 
-def test_explicit_inputs_in_subdirectories(renku_cli, client):
+def test_explicit_inputs_in_subdirectories(client, runner):
     """Test explicit inputs that are in sub-dirs are made accessible"""
-
     # Set up a script with hard dependency
-    renku_cli("run", "--no-output", "mkdir", "foo")
-    renku_cli("run", "echo", "some changes", stdout="foo/bar")
-    renku_cli("run", "echo", "cat foo/bar", stdout="script.sh")
+    assert 0 == runner.invoke(cli, ["run", "--no-output", "mkdir", "foo"]).exit_code
+    assert 0 == runner.invoke(cli, ["run", "echo", "some changes"], stdout="foo/bar").exit_code
+    assert 0 == runner.invoke(cli, ["run", "echo", "cat foo/bar"], stdout="script.sh").exit_code
 
-    exit_code, _ = renku_cli("run", "--input", "foo/bar", "--input", "script.sh", "sh", "script.sh", stdout="output")
-    assert 0 == exit_code
+    result = runner.invoke(
+        cli, ["run", "--input", "foo/bar", "--input", "script.sh", "sh", "script.sh"], stdout="output"
+    )
+    assert 0 == result.exit_code
```

Comment:
We should follow the `assert 0 == result.exit_code, format_result_exception(result)` pattern here